src/groupeddataframe/utils.jl

```diff
@@ -5,11 +5,13 @@ function hashrows_col!(h::Vector{UInt},
                        v::AbstractVector{T},
                        rp::Nothing,
                        firstcol::Bool) where T
-    @inbounds for i in eachindex(h)
-        el = v[i]
-        h[i] = hash(el, h[i])
-        if length(n) > 0
-            n[i] |= ismissing(el)
+    @splittasks 1_000_000 for i in eachindex(h)
+        @inbounds begin
+            el = v[i]
+            h[i] = hash(el, h[i])
+            if length(n) > 0
+                n[i] |= ismissing(el)
+            end
```

Comment:
Changing these lines to:
```
 if eltype(v) >: Missing && length(n) > 0
        n .|= ismissing.(v)
    end
```
test and moving it oudide might make sense in general. What do you think?

Reply:
Also I have confirmed that only this line causes problem, ie. `n[i] |= ismissing(el)` does not get executed properly. I will report the details.

Reply:
it happens if we work with `CategoricalVector`.

Reply:
Let us keep it as is for now - it is not likely a big difference anyway